build/nuget/build.ps1

```diff
@@ -21,7 +21,7 @@ if(!$Repo) {
 
 $version = "0.0.1"
```

Comment:
That is a different version number that I don't really understand. It is what is baked into vsimporter and xib2nib only at the moment. It is independent of the nuget version but we could later bring them in line. Remember that the package @pradipd  is working on here is strictly prototype for now. We will start over the versioning with the new packages I am working on.